core/src/main/java/com/github/phantazmnetwork/core/config/processor/ItemStackConfigProcessors.java

```diff
@@ -28,25 +24,24 @@ private ItemStackConfigProcessors() {
      */
     public static @NotNull ConfigProcessor<ItemStack> snbt() {
         return new ConfigProcessor<>() {
+
+            private final ConfigProcessor<NBT> nbtCompoundProcessor = MinestomConfigProcessors.nbt();
```

Comment:
thought i commented this before but guess not: this should be static

Reply:
why would it matter?

Reply:
doesn't really but it's a convention we've followed thus far